jee-commons/src/main/java/biz/paluch/jee/commons/BeanLookup.java

```diff
@@ -10,9 +10,14 @@
 public final class BeanLookup {
 
     /**
-     * Default JNDI name of the BeanManager.
+     * Default JNDI names of the BeanManager.
      */
-    public static final String BEANMANAGER_JNDI_NAME = "java:comp/BeanManager";
+    public static final String[] BEANMANAGER_JNDI_NAMES = {"java:comp/BeanManager", "java:comp/env/BeanManager"};
```

Comment:
What do you think about changing this to List<String>? That way users are able to alter the order and content of the JNDI names they would like to look up.